components/RadioButton.vue

```diff
@@ -1,26 +1,46 @@
 <template>
   <fieldset class="usa-fieldset">
-    <legend class="usa-legend usa-legend">{{ label }}</legend>
-    <div
-      v-for="value in values"
-      :key="value"
-      class="usa-radio tablet:margin-left-05"
-    >
-      <input
-        :id="`${uniqueId}-${criteriaKey}-${value}`"
-        class="usa-radio__input"
-        type="radio"
-        :name="`${uniqueId}-${criteriaKey}-${value}`"
-        :value="value"
-        :checked="response === value"
-        @change="updateEligibilitySelected"
-      />
-      <label
-        class="usa-radio__label tablet:margin-top-1"
-        :for="`${uniqueId}-${criteriaKey}-${value}`"
-        >{{ value }}</label
+    <legend class="usa-legend usa-legend" :class="selectedStyle">
+      {{ label }}
+    </legend>
+    <template v-for="(value, index) in values">
+      <div
+        v-if="location === 'benefit-card' && index === 0"
+        :key="`${value}-${naUniqueId}`"
+        class="usa-radio tablet:margin-left-05"
```

Comment:
what is the naUniqueId for??  do we need a different uniqueId if we already have one?

Reply:
It is used for the `not applicable` option that we are adding (only ) to the benefit card. We don't want it added from the content files so we are manually adding a way to unselect the option within in the benefit card.